taxonomy: move processing from ingredients (#11811)
openfoodfacts/openfoodfacts-server: taxonomies/ingredients_processing.txt

##################### THE RESULT OF THE PROCESS ##############

en: whole, and whole
bg: цял, цяло, цяла, цели
ca: sencer, sencera, sencers, senceres

> da: hele, sød

Reviewer: “sød” does not make sense here.

es: en polvo, polvo de
et: pulber
fa: پودر
fi: jauhe, jauhetta

> fr: en poudre, poudre de, poudre d', semoule de, semoule d', semoule, en semoule, poudre, à base de poudre de, à base de poudre de lait, poudres de

Reviewer 2: The milk should not be there I think.

```suggestion
fr: en poudre, poudre de, poudre d', semoule de, semoule d', semoule, en semoule, poudre, à base de poudre de poudres de
```

pt: Xarope
sv: sirap

# anhydrous: No water, dehydrated: Water removed, dried: Less water.
en: anhydrous
ar: لامائي
ca: Anhidre
da: Vandfri

> es: 	Anhidro
> eu: 	Anhidro
> fa: 	انیدروز
> fr: 	Anhydre
> id: 	Anhidrat
> it: 	Anidro
> nl: 	Watervrij
> pt: 	Anidro
> vi: 	Khan
> zh: 	无水物

Reviewer: ```suggestion
es: Anhidro
eu: Anhidro
fa: انیدروز
fr: Anhydre
id: Anhidrat
it: Anidro
nl: Watervrij
pt: Anidro
vi: Khan
zh: 无水物
```

nb: kondensert
nl: gecondenseerde
pl: skondensowane
pt: condensada, condensado
ro: condensat
ru: сгущенное, сгущённое
sl: kondenzirano
sv: kondenserad
th: ข้น
tl: na kondensada
tr: Kondense
tt: Куертылган
uk: згущене
vi: đặc
zh: 煉

# description:en:LOW-FAT MILK is a 1% fat milk, sold in the United States
# description:en:Skimmed milk (British English) is made when all the cream is removed from whole milk. It tends to contain around 0.1% fat. Láit écrémé has a maximum of 0.5% fat.
# description:en:Skim milk (American English), is made when all the cream is removed from whole milk. It tends to contain around 0% fat.
# Nonfat, skim milk and skimmed milk are often used as synonyms and the
# regulatory differences are small, with the average being between 0.0% and
# 0.2% fat. We are thus using only one taxonomy entry for all milks with
# a maximum of 0.5% fat

en: skimmed, skim, non-fat, nonfat, fat-free, low-fat, lowfat, defatted, 1% fat, with 0.5% milk fat
ar: منزوع الدسم
bg: обезмаслено, обезмаслно, обезмаслен, обезмаслена, обезмаслени
ca: desnatada, descremada, desgreixat, desgreixada, desgreixats, desgreixades, sense greix, baixa en greixos
cs: odstředěné, odstředěné

> da: skummet

Reviewer: “skummet” _can_ refer to skimming (which implies removing the froth), but generally refers to the result of any process that creates froth (e.g., whipping cream or frothing coffee).

```suggestion
```

de: Mager, entrahmtes, entrahmte, entfettet, entfettete, entfetteter, entfettetes, entfetteten, entfettetem, fettarme
el: αποβουτυρωμένο
es: desnatada, descremada, desengrasado, desengrasada, desgrasado, desgrasada, sin grasas, baja en grasa
et: koorimata, kooritud
fa: خشک بدون چربی
fi: rasvaton, rasvatonta, Vähärasvainen
fr: écrémé, maigre, dégraissé, dégraissée, dégraissés, dégraissées
gl: desnatado
he: רזה
hr: obrano, obrane, obrani
hu: sovány, zsírtalanított, fölözött, alacsony zsírtartalmú
id: skim
is: undanrennur
it: scremato, magro, sgrassato, sgrassata, senza grassi, magro
ja: 脱脂
lt: nugriebtas, nugriebto, lieso
lv: vaj, vāj
mk: обрано
ms: skim

> nb: skummet
> nl: mager, magere, ontvet, ontvette, tapte, onder, afgeroomde, vetvrije
> no: skummet

Reviewer: I don’t know Norwegian, but based on its proximity to Danish (and Swedish), I would assume the comment for Danish also applies here.

```suggestion
nl: mager, magere, ontvet, ontvette, tapte, onder, afgeroomde, vetvrije
```

pl: odtłuszczone, chude, o obniżonej zawartości tłuszczu, odtłuszczony, odtłuszczona, odtłuszczonego
pt: magro, desnatado, desengordurado, desengordurada
ro: degresat
ru: обезжиренноё, и обезжиренноё, обезжиренный, обезжиренное, безжиренное, и обезжиренное
sk: odstredené
sl: posneto
sq: i skremuar

> sv: skum, lätt

Reviewer: In addition to “skum” probably referring to anything that produces froth (see comment on `da` entry), “lätt” doesn’t necessarily refer to reduced fat content, but can also be used as a descriptor for any reduction of caloric content.

```suggestion
```

th: ขาดมันเนย
tr: yağsız, kaymağı alınmış
uk: Знежирене
vi: gầy
zh: 脱脂

< en:skimmed
en: semi-skimmed, partially defatted, reduced-fat, partly skimmed, partially skimmed, part skim
bg: Полуобезмаслено, частично обезмаслен, частично обезмаслено
ca: parcialment desnatada, semidesnatada, parcialment desgreixat, parcialment desgreixada, parcialment desgreixats, parcialment desgreixades
de: teilentrahmte, teilentfettet, teilentfettete, teilentfettetes, teilentfetteter, teilentfetteten, teilentfettetem, halbentrahmte
es: parcialmente desnatada, semidesnatada, semidescremada, parcialmente descremada
fi: kevyt
fr: semi-écrémé, demi-écrémé, partiellement écrémé
hr: poluobrano, djelomično obrano, djelimično obrano
hu: zsírszegény, félzsíros

> is: létt
> it: semi-magro, parzialmente scremato, semiscremato
> nb: lett

Reviewer: I don’t know Icelandic _or_ Norwegian, but again, with their proximity to Danish and Swedish, I would assume that `l[eé]tt` also means “light” similar to Danish “let” and Swedish “lätt”. “Light” can refer to not just a reduction of fat content, but to any reduction of calories.
```suggestion
it: semi-magro, parzialmente scremato, semiscremato
```

#fr:germe, gluten, amidon
#hr:klica, gluten, škrob

en: condensed
af: Kondens
ar: المبخر
az: Qatılaşdırılmış
be: згушчанае
bg: кондензирано
ca: condensada
cs: kondenzované

> da: kondenserad, kondenseret

Reviewer: “kondenserad” is not Danish. Is this a Swedish ingredient list that got conflated with Danish at some point?
```suggestion
da: kondenseret
```

fa: خشک بدون چربی
fi: rasvaton, rasvatonta, Vähärasvainen
fr: écrémé, maigre, dégraissé, dégraissée, dégraissés, dégraissées
gl: desnatado
he: רזה
hr: obrano, obrane, obrani
hu: sovány, zsírtalanított, fölözött, alacsony zsírtartalmú
id: skim
is: undanrennur
it: scremato, magro, sgrassato, sgrassata, senza grassi, magro
ja: 脱脂
lt: nugriebtas, nugriebto, lieso
lv: vaj, vāj
mk: обрано
ms: skim
nb: skummet
nl: mager, magere, ontvet, ontvette, tapte, onder, afgeroomde, vetvrije
no: skummet
pl: odtłuszczone, chude, o obniżonej zawartości tłuszczu, odtłuszczony, odtłuszczona, odtłuszczonego
pt: magro, desnatado, desengordurado, desengordurada
ro: degresat
ru: обезжиренноё, и обезжиренноё, обезжиренный, обезжиренное, безжиренное, и обезжиренное
sk: odstredené
sl: posneto
sq: i skremuar
sv: skum, lätt
th: ขาดมันเนย
tr: yağsız, kaymağı alınmış
uk: Знежирене
vi: gầy
zh: 脱脂

< en:skimmed
en: semi-skimmed, partially defatted, reduced-fat, partly skimmed, partially skimmed, part skim
bg: Полуобезмаслено, частично обезмаслен, частично обезмаслено
ca: parcialment desnatada, semidesnatada, parcialment desgreixat, parcialment desgreixada, parcialment desgreixats, parcialment desgreixades

> da: let

Reviewer: This doesn’t make sense here. “Let” can also be used for a number of other products/processes that has fat (or calories) reduced.
```suggestion
```

de: teilentrahmte, teilentfettet, teilentfettete, teilentfettetes, teilentfetteter, teilentfetteten, teilentfettetem, halbentrahmte
es: parcialmente desnatada, semidesnatada, semidescremada, parcialmente descremada
fi: kevyt
fr: semi-écrémé, demi-écrémé, partiellement écrémé
hr: poluobrano, djelomično obrano, djelimično obrano
hu: zsírszegény, félzsíros
is: létt
it: semi-magro, parzialmente scremato, semiscremato
nb: lett
nl: halfvolle, gedeeltelijk afgeroomde
pl: częściowo odtłuszczone, częściowo odtłuszczony, częściowo odtłuszczona, częściowo odtłuszczonego, półtłusty, półtłuste
pt: meio gordo, parcialmente desengordurado, parcialmente desengordurada, parcialmente desnatado
ro: demidegresat

> sv: mella

Reviewer: Also no :) (Probably not for `is` and `nb` either.)
```suggestion
```

fr: non pasteurisé
pl: niepasteryzowana, niepasteryzowane, niepasteryzowany
pt: não pasteurizado, não pasteurizados, não pasteurizada, não pasteurizadas

> sv: opastöriserad

Reviewer: I’m glad this is getting fixed! Thank you! ❤️

pl: bez soli, niesolone
pt: sem sal, não salgado

> # sweetened is not a processing as it means an extra ingredient. 
> # It changes an ingredients into a 'compound' ingredient.
> en: sweetened, sugared, with sugar
> bg: подсладено, подсладен, подсладена, подсладени, захаросано, захаросан, захаросана, захаросани
> ca: ensucrada, amb sucre, dolc de
> cs: slazené
> da: sødet, sukret
> de: gesüßt, gesüßte, gezuckert, gezuckerte, gezuckerter, gezuckertes
> el: ζαχαρούχο
> es: endulzado, endulzada, endulzados, endulzadas, azucarado, azucarada, azucarados, azucaradas, dulce de, azucadara
> et: magustatud
> fi: makeutettu, makeutetut, sokeroitu, sokeroidut, sokeroitua, sokeroituja
> fr: sucré, sucrée, sucrés, sucrées, sucree, sucrees
> hr: zaslađene, zaslađeno, zasladeno
> hu: édesített, cukrozott
> it: addolcito, addolciti, addolcita, addolcite, zuccherato, zuccherati, zuccherata, zuccherate
> ja: 加糖
> lt: saldintas
> lv: saldināts, saldinats
> nb: sukkerholdig, sukret, søt
> nl: gezoet, gezoete, gesuikerd, gesuikerde, met suiker
> pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
> pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas, açucarado, açucarada, açucarados, açucaradas
> ro: indulcit
> ru: с сахаром
> sl: sladkano
> sv: sötad, söt, sockrad
> th: หวาน
> 
> < en:concentrated
> < en:sweetened
> en: concentrate for sweetening
> hr: koncentrat za zaslađivanje
> 
> # same comment as sweetened

Reviewer: ```suggestion
en: sweetened
bg: подсладено, подсладен, подсладена, подсладени
ca: dolc de
cs: slazené
da: sødet
de: gesüßt, gesüßte
el: ζαχαρούχο
es: endulzado, endulzada, endulzados, endulzadas, dulce de
et: magustatud
fi: makeutettu, makeutetut
fr: sucré, sucrée, sucrés, sucrées, sucree, sucrees
#better not add fr:sucre and fr:sucres
hr: zaslađene, zaslađeno, zasladeno
hu: édesített
it: addolcito, addolciti, addolcita, addolcite
lt: saldintas
lv: saldināts, saldinats
nb: søt
nl: gezoet, gezoete
pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas
ro: indulcit, îndulcite
sl: sladkano
sv: sötad, söt
th: หวาน

< en:sweetened
en: sugared, with sugar
bg: захаросано, захаросан, захаросана, захаросани
ca: ensucrada, amb sucre
da: sukret, med sukker
de: gezuckert, gezuckerte, gezuckerter, gezuckertes
es: azucarado, azucarada, azucarados, azucaradas, azucadara
fi: sokeroitu, sokeroidut, sokeroitua, sokeroituja
hu: cukrozott
it: zuccherato, zuccherati, zuccherata, zuccherate
ja: 加糖
nb: sukret, sukkerholdig
nl: gesuikerd, gesuikerde, met suiker
pt: açucarado, açucarada, açucarados, açucaradas
ru: с сахаром
sv: sockrad
```

I don’t know if French “sucré” refers to general sweetening or if it’s specifically referring to sweetening with sugar, but I lumped it in here with general sweetening since that matches the equivalent `en:unsweetened` translations/synonyms. For the rest of the languages, as best I can tell, the first section is generic “sweetening” while the second is specifically sweetening using sugar.

pl: bez soli, niesolone
pt: sem sal, não salgado

# sweetened is not a processing as it means an extra ingredient. 

> # It changes an ingredients into a 'compound' ingredient.

Reviewer: I think this is going beyond the scope of the PR. If this is a distinction we want to make going forward, I think it deserves its own PR with its own discussion. It feels like overreach to group it in with “move processing from ingredients to processing”.

> # en: sweetened, sugared, with sugar

Reviewer: I said this in a previous comment, but sweetening and sugaring are two different (if related) processes. It won’t matter (for this file) if we end up move these out of processing, but, as per my other comment, I think that’s for a different PR and a different discussion. :)

# bg: подсладено, подсладен, подсладена, подсладени, захаросано, захаросан, захаросана, захаросани
# ca: ensucrada, amb sucre, dolc de
# cs: slazené
# da: sødet, sukret
# de: gesüßt, gesüßte, gezuckert, gezuckerte, gezuckerter, gezuckertes
# el: ζαχαρούχο
# es: endulzado, endulzada, endulzados, endulzadas, azucarado, azucarada, azucarados, azucaradas, dulce de, azucadara
# et: magustatud
# fi: makeutettu, makeutetut, sokeroitu, sokeroidut, sokeroitua, sokeroituja
# fr: sucré, sucrée, sucrés, sucrées, sucree, sucrees
# hr: zaslađene, zaslađeno, zasladeno
# hu: édesített, cukrozott
# it: addolcito, addolciti, addolcita, addolcite, zuccherato, zuccherati, zuccherata, zuccherate
# ja: 加糖
# lt: saldintas
# lv: saldināts, saldinats
# nb: sukkerholdig, sukret, søt
# nl: gezoet, gezoete, gesuikerd, gesuikerde, met suiker
# pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
# pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas, açucarado, açucarada, açucarados, açucaradas
# ro: indulcit
# ru: с сахаром
# sl: sladkano
# sv: sötad, söt, sockrad
# th: หวาน

< en:concentrated

> # < en:sweetened

Reviewer: See comment on `en: sweetened`.

```suggestion
< en:sweetened
```

# en:concentrate for sweetening
# hr: koncentrat za zaslađivanje

> # same comment as sweetened

Reviewer: also same comment as sweetened :wink:

pl: bez soli, niesolone
pt: sem sal, não salgado

> en: sweetened, sugared, with sugar

Reviewer: `en:sugared,with sugar` should probably be added on their own. `en:sweetened` can also mean sweetening by using a number of things/methods other than adding sugar (fruits, stevia, various artificial sweeteners, increasing fat percentage, …).

Reviewer 3: I would remove this from processing

Reviewer: I think the discussion about whether this should be in processing or not is beyond the scope of this PR (as [I’ve mentioned in other comments](https://github.com/openfoodfacts/openfoodfacts-server/pull/11811/files#r2116140972))—and while this _is_ part of `_processing.txt`, conflating “sweetened” and “sugared” seems like a loss of information.

(I think this is also the only thing I have left I would like to see changed about this PR. Very happy to see this merged in when these two are no longer being combined!)

bg: подсладено, подсладен, подсладена, подсладени, захаросано, захаросан, захаросана, захаросани
ca: ensucrada, amb sucre, dolc de
cs: slazené

> da: sockrad, sukret

Reviewer: “sockrad” is Swedish. “sødet” or maybe “forsødet” are direct translations of “sweetened”. Not sure which is more common.
```suggestion
da: sødet, sukret
```

de: gesüßt, gesüßte, gezuckert, gezuckerte, gezuckerter, gezuckertes
el: ζαχαρούχο
es: endulzado, endulzada, endulzados, endulzadas, azucarado, azucarada, azucarados, azucaradas, dulce de, azucadara
et: magustatud
fi: makeutettu, makeutetut, sokeroitu, sokeroidut, sokeroitua, sokeroituja
fr: sucré, sucrée, sucrés, sucrées, sucree, sucrees
hr: zaslađene, zaslađeno, zasladeno
hu: édesített, cukrozott
it: addolcito, addolciti, addolcita, addolcite, zuccherato, zuccherati, zuccherata, zuccherate
ja: 加糖
lt: saldintas
lv: saldināts, saldinats
nb: sukkerholdig, sukret, søt
nl: gezoet, gezoete, gesuikerd, gesuikerde, met suiker
pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas, açucarado, açucarada, açucarados, açucaradas
ro: indulcit
ru: с сахаром
sl: sladkano

> sv: sötad, söt, sockrad

Reviewer: If this PR as a whole goes through as it is, this will lead to some Swedish whole milks now being categorised by OFF as `en:sweetened + en:milk` despite there being no sweetener added. :)
